code/modules/client/preference_setup/occupation/occupation.dm

```diff
@@ -82,8 +82,10 @@
 			index = 0
 
 		. += "<tr bgcolor='[job.selection_color]'><td width='60%' align='right'>"
+
 		var/rank = job.title
 		lastJob = job
+		. += "<a href='?src=\ref[src];job_info=[rank]'>"
```

Comment:
You've added an opening `<a>` tag here.  May want to consider adding a matching closing `</a>` tag after the rank everywhere below so it generates valid HTML.